modules/ossm-observability-visual.adoc

```diff
@@ -5,34 +5,34 @@ This module is included in the following assemblies:
 ////
 
 [id="ossm-observability-visual_{context}"]
-= Working with data in the Kiali console
+= Viewing service mesh data in the Kiali console
 
-From the *Graph* menu in the Kiali console, you can use the following graphs and viewing tools to gain deeper insights about data that travels through your service mesh. These tools can help you identify problems with services or workloads.
+The Kiali Graph offers a powerful visualization of your mesh traffic. The topology combines real-time request traffic with your Istio configuration information to present immediate insight into the behavior of your service mesh, letting you quickly pinpoint issues. Multiple Graph Types let you visualize traffic as a high-level service topology, a low-level workload topology, or as an application-level topology.
```

Comment:
Curious as to whether Graph Types should be capitalized here. Just a thought.

Reply:
They're not titles, so no.